wire-ios/Wire-iOS/Sources/UserInterface/CustomAppLock/Setup/PasscodeSetupViewController.swift

```diff
@@ -187,21 +193,27 @@ final class PasscodeSetupViewController: UIViewController {
 
     private func createConstraints() {
 
-        [contentView, stackView].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
+        [scrollView, contentView, stackView].forEach { $0.translatesAutoresizingMaskIntoConstraints = false }
 
-        let widthConstraint = contentView.createContentWidthConstraint()
+        let heightConstraint = contentView.heightAnchor.constraint(greaterThanOrEqualTo: scrollView.heightAnchor)
 
         let contentPadding: CGFloat = 24
 
         NSLayoutConstraint.activate([
+            // scroll view
+            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            heightConstraint,
+
             // content view
-            widthConstraint,
             contentView.widthAnchor.constraint(lessThanOrEqualToConstant: 375),
-            contentView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
-            contentView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
-            contentView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            contentView.leadingAnchor.constraint(greaterThanOrEqualTo: view.leadingAnchor, constant: contentPadding),
-            contentView.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor, constant: -contentPadding),
+            contentView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            contentView.leadingAnchor.constraint(greaterThanOrEqualTo: scrollView.leadingAnchor, constant: contentPadding),
+            contentView.trailingAnchor.constraint(lessThanOrEqualTo: scrollView.trailingAnchor, constant: -contentPadding),
+            contentView.centerXAnchor.constraint(equalTo: scrollView.centerXAnchor),
```

Comment:
optional: Would it be much effort to use the [contentLayoutGuide](https://developer.apple.com/documentation/uikit/uiscrollview/2865870-contentlayoutguide) property?